src/error.rs

```diff
@@ -40,7 +40,7 @@ pub type Result<T> = std::result::Result<T, Error>;
 /// [`ErrorKind`](enum.ErrorKind.html) is wrapped in an `Arc` to allow the errors to be
 /// cloned.
 #[derive(Clone, Debug, Error)]
-#[error("{kind}")]
+#[error("Kind: {kind}, labels: {labels:?}")]
```

Comment:
currently we omitted the labels from the Display implementations so I added that. I currently use the Display impls to generate the string representations of errors to attach to "command failed" events. (tracing events only support attaching a few primitive types like strings and various numbers.)

otherwise, I wanted to avoid going down a rabbit hole of hashing out the ideal string representations for every error type in the driver so I just left those as-is. since they are just string representations I don't think there are any semver concerns, and if we find issues with them in the future we could improve them as we go.